src/pages/Home.js

```diff
@@ -0,0 +1,141 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import './Home.css'
+import PokemonTile from "../components/pokemonTIle/PokemonTile";
+import FlipThroughButton from "../components/flipthroughbutton/FlipThroughButton";
+import LimitSelector from "../components/limitselector/LimitSelector";
+import pokemonLogo from '../assets/pokemonlogo.png'
+
+
+function Home(props) {
+
+    const ENDPOINT = 'https://pokeapi.co/api/v2/'
+    const [error, toggleError] = useState(false)
+    const [isLoading, toggleIsLoading] = useState(false)
+    // states to store the offset and limit for the request
+    const [pokemonLimit, setPokemonLimit] = useState(20)
+    const [pokemonOffset, setPokemonOffset] = useState(0)
+    const [reachedEnd, setReachedEnd] = useState('')
+    const [upperLimitShown, setUpperLimitShown] = useState(0)
+    // empty state to store the data from the request
+    const [pokemonGroupData, setPokemonGroupData] = useState({})
+
+
+    // requests a set of pokemon to be displayed on the page
+    async function fetchAllPokemon() {
+        toggleIsLoading(true)
+        try {
+            // resets error back to initial state
+            toggleError(false)
+            // gets the data from the request and stores it in the state
+            setPokemonGroupData(
+                (await axios.get(
+                    `${ENDPOINT}pokemon?offset=${pokemonOffset}&limit=${pokemonLimit}`
+                )).data
+            )
+
+        } catch (e) {
+            toggleError(true)
+            console.error(e)
+        }
+        toggleIsLoading(false)
+    }
+
+    // function to decrease or increase the offset depending on the button pressed
+    function handleClick(variant) {
+        if (variant === 'next') {
+            setPokemonOffset(pokemonOffset + pokemonLimit)
+        }
+        if (variant === 'previous') {
+            // checks whether the next step will take the user over the end of the list
+            pokemonOffset - pokemonLimit < 0 ? setPokemonOffset(0) :
+                setPokemonOffset(pokemonOffset - pokemonLimit)
+
+        }
+    }
+
+    function CheckLimit() {
```

Comment:
Functienamen camelCase schrijven
```suggestion
    function checkLimit() {
```